manifests/manifests_test.go

```diff
@@ -0,0 +1,47 @@
+package manifests_test
+
+import (
+	"bytes"
+	"fmt"
+	"maistra.io/api/manifests"
+	"testing"
+)
+
+func TestGettingManifestDirEntries(t *testing.T) {
+	manifests, err := manifests.GetManifests()
```

Comment:
You can just call `GetManifests()` as the test is in the same package

Reply:
As above: the package of the test is `manifests_test`